src/plugins/Dashboard/index.js

```diff
@@ -210,9 +210,12 @@ module.exports = class DashboardUI extends Plugin {
     document.body.classList.add('is-UppyDashboard-open')
     document.body.style.top = `-${this.savedDocumentScrollPosition}px`
 
+    this.updateDashboardElWidth()
+    this.setFocusToFirstNode()
+
     // timeout is needed because yo-yo/morphdom/nanoraf; not needed without nanoraf
```

Comment:
lets get rid of this outdated comment! might be confusing otherwise

Reply:
I thought it was actually useful this time, because it reminded of what was wrong with the previous code, but all right :)